backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/modelvalidator/C_OLCand.java

```diff
@@ -283,7 +284,8 @@ private void updateHandoverLocationOverride(@NonNull final I_C_OLCand olCand)
 		}
 		else
 		{
-			org.compiere.model.I_C_BPartner_Location handOverLocation = handoverRelation.getC_BPartnerRelation_Location();
+			org.compiere.model.I_C_BPartner_Location handOverLocation = Services.get(IBPartnerDAO.class).getBPartnerLocationById(BPartnerLocationId.ofRepoId(handoverRelation.getC_BP_Relation_ID(), handoverRelation.getC_BPartnerRelation_Location_ID()));
```

Comment:
consider breaking this line into multiple lines so it gets much more readable and clean.